app/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
     implementation Dependencies.design
     implementation Dependencies.constraint_layout
     implementation Dependencies.appCompat
+    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.0.0'
```

Comment:
I think the PR had conflicts in commits right, Can you check it again plz?

Reply:
I checked it, there were no conflicts.
Could you please specify?

Reply:
@anna4j  jarRepositories.xml didn't removed yet, I cloned the project and no conflicts showed.